process_request/routines.py

```diff
@@ -29,36 +32,40 @@ def get_data(self, item):
                         username=settings.ARCHIVESSPACE["username"],
                         password=settings.ARCHIVESSPACE["password"],
                         repository=settings.ARCHIVESSPACE["repo_id"])
-        obj = aspace.client.get(item)
+        obj = aspace.client.get(item, params={"resolve": ["resource::linked_agents", "ancestors", "top_container", "top_container::container_locations"]})
         if obj.status_code == 200:
-            return obj.json()
+            as_data = {}
```

Comment:
I think it might be a bit cleaner to NOT set an empty dict to a variable, but instead to just do:

```
return {
  "creator": get_collection_creator(item_collection),
  "restrictions": "TK,
...
}
```
It's a bit of a personal preference thing but I feel like it might clarify things a bit in the code.